src/seedu/addressbook/ui/MainWindow.java

```diff
@@ -50,6 +53,9 @@ void onCommand(ActionEvent event) {
                 exitApp();
                 return;
             }
+            else if(isThemeCommand(result)) {
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. missing spaces after `if`.